src/query-builder/SelectQueryBuilder.ts

```diff
@@ -1616,6 +1619,16 @@ export class SelectQueryBuilder<Entity> extends QueryBuilder<Entity> implements
                 } else if (driver instanceof SqlServerDriver) {
                     return "";
 
+                } else {
+                    throw new LockNotSupportedOnGivenDriverError();
+                }
+            case "dirty_read":
+                if (driver instanceof SqlServerDriver) {
+                    return " WITH (NOLOCK)";
+                    
+                } else if (driver instanceof MysqlDriver || driver instanceof PostgresDriver || driver instanceof OracleDriver) {
+                    return "";
```

Comment:
Why just ignore here? I doubt it's the default setting for those drivers.

Reply:
Hello, it's my first climb so I apologize in advance for any possible errors. Searching the internet I found this about equivalents to this instruction in other databases:

MySQL:
When WITH (nolock) is used in SQL Server to perform operation, it does not place shared lock (S) and also does not honor exclusive locks on table.  The WITH (nolock) table hint is equivalent to READ UNCOMMITTED also known as "dirty read", it is the lowest level of isolation. If you specify table hint then it will override the current isolation level.
MySQL default isolation level is REPEATABLE READ which means locks will be placed for each operation but multiple connections can read data concurrently with out making table unavailable during read. MySQL database server does not support changing the default isolation mode as a hint like SQL Server.

Oracle:
Oracle by default provides a read-consistent view of data.  The upshot of this design is that readers never block writers.

PostgreSQL:
PostgreSQL uses MVCC to minimize lock contention in order to allow for reasonable performance in multiuser environments. Readers do not conflict with writers nor other readers. The closest thing would be the NOWAIT

It's what I found about it.

Reply:
I still don't understand why we just pass empty sting in some drivers and not throwing `LockNotSupportedOnGivenDriverError`.
If driver doesn't support this it's fine but I don't think we should throw an error, not just ignore this - otherwise people using it will think that everything is working as expected while underneath typeorm would just ignore some unsupported features.

Reply:
I think it would be better to throw in error, although really this kind of cases do not think it is an error, it would be more informative that the driver does not have that functionality. What do you think?

Reply:
I just realized that this part of the code should not be set, it is enough with the change of line 1388. I delete this part of the code.